src/Chirp.Razor/Program.cs

```diff
@@ -0,0 +1,25 @@
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddRazorPages();
+builder.Services.AddSingleton<ICheepService, CheepService>();
+
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (!app.Environment.IsDevelopment())
+{
+    app.UseExceptionHandler("/Error");
+    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+    app.UseHsts();
+}
+
+app.UseHttpsRedirection();
```

Comment:
Temporarily this seems to cause a slight issue as we use a HTTP port instead of HTTPS. This can be fixed in launchSeetings.json if needed.

Reply:
This has been resolved in commit: ae29772